### PR TITLE
Unittest

### PR DESCRIPTION
**Pull request by fjebaker**
Added unittests for the *non-async* `covid_bot/cogs/stats.py` class methods, which invloved a minimal pytest environment for testing future cogs. Tests focus on the sunny day behaviour of our methods; provided the API is working as intended, the unit tests ensure our functions will behave correctly also.

Added tests for the API and relevant API calls that we make -- all but the healthcheck is skipped, unless you add the `--api` flag when running unit tests.

Tests should all be runnable with just
```
pytest
```
from the route directory.

### CODE REVIEW

**Comment by fjebaker** [comment]
@jwiggins merge if you're okay with the changes :)

File: test/conftest.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import pytest
+from unittest.mock import MagicMock
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--api", action="store_true", help="runs the tests marked @pytest.mark.api"
+    )
```
Comment (jwiggins) [inline test/conftest.py:8]: For politeness reasons?

**Review by jwiggins — APPROVED** [review]
Is this less code or just cleaner formatting?

(looks like you're using flake8 now? 👍)

File: test/conftest.py (hunk 1 above)
Comment (fjebaker) [reply test/conftest.py:8]: That, and also the api tests take considerably longer than all the other tests combined.

**Pull request merged**